Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -22,6 +22,17 @@ namespace
 
 namespace AssetProcessor
 {
+
+    void AssetImporterPathsVisitor::Visit([[maybe_unused]] AZStd::string_view path, AZStd::string_view, AZ::SettingsRegistryInterface::Type,
+        AZStd::string_view value)
+    {
+        auto found = value.find('.');
+        if (found != AZStd::string::npos)
+        {
+            m_supportedFileExtensions.emplace_back(value.substr(found + 1));
+        }
```

Comment:
Should there be an else here, an assert, or a warning? I'd lean toward an else and just take the extension as-is, assume it's without a period in that case, so "fbx" instead of ".fbx"

Reply:
good idea, i was add one